modules/widgets/social-icons.php

```diff
@@ -388,277 +416,256 @@ public function get_svg_icon( $args = array() ) {
-				'url'   => 'amazon.de',
-				'icon'  => 'amazon',
-				'label' => 'Amazon',
-			),
-			array(
-				'url'   => 'amazon.it',
-				'icon'  => 'amazon',
-				'label' => 'Amazon',
-			),
-			array(
-				'url'   => 'amazon.nl',
-				'icon'  => 'amazon',
-				'label' => 'Amazon',
-			),
-			array(
-				'url'   => 'amazon.es',
-				'icon'  => 'amazon',
-				'label' => 'Amazon',
-			),
-			array(
-				'url'   => 'amazon.co',
-				'icon'  => 'amazon',
-				'label' => 'Amazon',
-			),
-			array(
-				'url'   => 'amazon.ca',
-				'icon'  => 'amazon',
-				'label' => 'Amazon',
-			),
-			array(
-				'url'   => 'amazon.com',
+				'url'   => array(
+					'amazon.cn',
+					'amazon.in',
+					'amazon.fr',
+					'amazon.de',
+					'amazon.it',
+					'amazon.nl',
+					'amazon.es',
+					'amazon.co',
+					'amazon.ca',
+					'amazon.com',
+				),
 				'icon'  => 'amazon',
 				'label' => 'Amazon',
 			),
 			array(
-				'url'   => 'apple.com',
+				'url'   => array( 'apple.com' ),
 				'icon'  => 'apple',
 				'label' => 'Apple',
 			),
 			array(
-				'url'   => 'itunes.com',
+				'url'   => array( 'itunes.com' ),
 				'icon'  => 'apple',
 				'label' => 'iTunes',
 			),
 			array(
-				'url'   => 'bandcamp.com',
+				'url'   => array( 'bandcamp.com' ),
 				'icon'  => 'bandcamp',
 				'label' => 'Bandcamp',
 			),
 			array(
-				'url'   => 'behance.net',
+				'url'   => array( 'behance.net' ),
 				'icon'  => 'behance',
 				'label' => 'Behance',
 			),
 			array(
-				'url'   => 'codepen.io',
+				'url'   => array( 'codepen.io' ),
 				'icon'  => 'codepen',
 				'label' => 'CodePen',
 			),
 			array(
-				'url'   => 'deviantart.com',
+				'url'   => array( 'deviantart.com' ),
 				'icon'  => 'deviantart',
 				'label' => 'DeviantArt',
 			),
 			array(
-				'url'   => 'digg.com',
+				'url'   => array( 'digg.com' ),
 				'icon'  => 'digg',
 				'label' => 'Digg',
 			),
 			array(
-				'url'   => 'discord.gg',
-				'icon'  => 'discord',
-				'label' => 'Discord',
-			),
-			array(
-				'url'   => 'discordapp.com',
+				'url'   => array( 'discord.gg', 'discordapp.com' ),
 				'icon'  => 'discord',
 				'label' => 'Discord',
 			),
 			array(
-				'url'   => 'dribbble.com',
+				'url'   => array( 'dribbble.com' ),
 				'icon'  => 'dribbble',
 				'label' => 'Dribbble',
 			),
 			array(
-				'url'   => 'dropbox.com',
+				'url'   => array( 'dropbox.com' ),
 				'icon'  => 'dropbox',
 				'label' => 'Dropbox',
 			),
 			array(
-				'url'   => 'etsy.com',
+				'url'   => array( 'etsy.com' ),
 				'icon'  => 'etsy',
 				'label' => 'Etsy',
 			),
 			array(
-				'url'   => 'facebook.com',
+				'url'   => array( 'facebook.com' ),
 				'icon'  => 'facebook',
 				'label' => 'Facebook',
 			),
 			array(
-				'url'   => '/feed/',
+				'url'   => array(
+					'/feed/',        // WordPress default feed url.
+					'/feeds/',       // Blogspot and others.
+					'/blog/feed',    // No trailing space WordPress feed, could use /feed but may match unexpectedly.
```

Comment:
Should this say "no trailing slash"?

Also, do we have a concrete example of where this entry would be useful?

Reply:
Yep, that should be "No trailing slash".

Reply:
done in b82da9ec4ddb0ca3198fa52be6fbdae6ae207999